tests/pytests/integration/modules/test_mac_xattr.py

```diff
@@ -0,0 +1,181 @@
+"""
+integration tests for mac_xattr
+"""
+
+import os
+
+import pytest
+
+pytestmark = [
+    pytest.mark.skip_if_binaries_missing("xattr"),
+    pytest.mark.slow_test,
+    pytest.mark.skip_if_not_root,
+    pytest.mark.skip_unless_on_darwin,
+]
+
+
+@pytest.fixture(scope="function")
+def setup_teardown_vars(salt_call_cli, tmp_path):
+    test_file = str(tmp_path / "xattr_test_file.txt")
+    no_file = str(tmp_path / "xattr_no_file.txt")
+
+    salt_call_cli.run("file.touch", test_file)
+
+    try:
+        yield test_file, no_file
+    finally:
+        if os.path.exists(test_file):
+            os.remove(test_file)
```

Comment:
```suggestion
@pytest.fixture(scope="function")
def setup_teardown_vars(salt_call_cli, tmp_path):
    test_file = tmp_path / "xattr_test_file.txt"
    no_file = str(tmp_path / "xattr_no_file.txt")

    test_file.touch()

    try:
        yield str(test_file), no_file
    finally:
        if test_file.exists():
            test_file.unlink()
```
Why are we going through salt to touch a file? Waste of time and resources...